docs/src/getting-started/installation.md

```diff
@@ -9,7 +9,7 @@ In this section, we will walk through the process of installing Starknet Foundry
 * [Installation](#installation)
   * [Contents](#contents)
   * [Requirements](#requirements)
-  * [Linux and MacOS](#linux-and-macos)
+  * [Linux and macOS](#linux-and-macos)
```

Comment:
https://en.wikipedia.org/wiki/MacOS

Reply:
🍎

Reply:
I was unsure if this should be capitalized or not. But this formatting works for me